l1-contracts/contracts/state-transition/chain-deps/facets/Mailbox.sol

```diff
@@ -230,8 +221,8 @@ contract MailboxFacet is ZkSyncHyperchainBase, IMailbox {
         );
         IL1SharedBridge(s.baseTokenBridge).bridgehubDepositBaseToken{value: msg.value}(
             s.chainId,
+            (IL1SharedBridge(s.baseTokenBridge).nativeTokenVault()).getAssetId(ETH_TOKEN_ADDRESS),
```

Comment:
Here we:

- read from hot storage
- called shared bridge and did a cold read
- then called a new contract (which is probably yet another cold load of contract)

I would probably suggest creating a constant for this value